bundle.md

```diff
@@ -1,6 +1,6 @@
-# Filesystem Bundle
+# <a name="filesystemBundle" />Filesystem Bundle
 
-## Container Format
+## <a name="containerFormat" />Container Format
```

Comment:
I don't think we need to add anchors to the headers, since both [GitHub][1] and Pandoc agree on the auto-generated `container-format` anchor (and likewise for other headers):

    $ curl -Ls https://github.com/opencontainers/runtime-spec/releases/download/v1.0.0-rc2/oci-runtime-spec-1.0.0-rc2.html | grep 'Container Format'
    <h2 id="container-format">Container Format</h2>
[1]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md#container-format